src/api/custom-resolvers/mutations/create-many-applications/index.ts

```diff
@@ -0,0 +1,101 @@
+import { CreateManyApplicationsVariables, Application, Context, ObjectType } from '../../../types';
+import getCountries from '../../../helpers/get-countries';
+import { mockApplicationCreationObject } from '../../../test-mocks/mock-application';
+
+/**
+ * createManyApplications
+ * Creates many applications.
+ * This is used by E2E tests only
+ * 1) Creates an empty object array with the number of elements passed in variables
+ * 2) Gets all countries and creates a buyer with the first country
+ * 3) Maps application creation object
+ * 4) Creates many applications
+ * 5) Creates reference numbers for applications and updated ReferenceNumber and Application tables
```

Comment:
Can we move step 2x (line 10) into new lines/steps please

Same with "update" part on step 5x

As they mention multiple things